docs/developer/app-store/apps/invoices.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Invoices
+sidebar_position: 4
+---
+
+import { AppMetadata } from "/components/AppMetadata/AppMetadata.tsx";
+
+# Invoices
+
+<AppMetadata
+  minSaleorVersion="3.10"
+  roadmapUrl="https://github.com/saleor/apps/labels/App%3A%20Invoices"
+  githubUrl="https://github.com/saleor/apps/tree/main/apps/invoices"
+/>
+
+## Introduction
+
+The Invoices app allows you to generate simple PDF invoice files for every order in your store.
+
+## Features
+
+Generates a PDF file when an invoice is requested (in Dashboard or with an API). Then attaches it to the order.
+
+The invoice shop address can be inherited from the [Shop](https://docs.saleor.io/docs/3.x/api-storefront/miscellaneous/objects/shop) settings or configured manually per channel.
```

Comment:
Docusaurus won't validate that link, please make it relative.

Reply:
done